From 1d03ef6d0e31a92e245a3ba2153487fe9cebfcde Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Tue, 28 Jul 2015 11:09:20 +0800
Subject: [PATCH 0035/5242] MLK-10447-01: dts: imx7d sdb: Add 74LV595 gpio
 expander node

commit  9c8ddb886619feb64cb6c9b18b0da4b178cd4973 from
https://source.codeaurora.org/external/imx/linux-imx.git

-74LV595 function compatible with 74HC595, add 74HC595
driver into imx7d sdb dts.

Signed-off-by: Sandor Yu <R01008@freescale.com>
Signed-off-by: Fugang Duan <B38611@freescale.com>

This was also added upstream, only change that remains is adding
registers-default on the gpio-expander node.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 7752692..8256b20 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -49,6 +49,7 @@
 			#gpio-cells = <2>;
 			reg = <0>;
 			registers-number = <1>;
+			registers-default = /bits/ 8 <0x74>; /* Enable PERI_3V3, SENSOR_RST_B and HDMI_RST*/
 			spi-max-frequency = <100000>;
 		};
 	};
-- 
1.7.9.5

